libs/island-ui/core/src/lib/Toast/Toast.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import * as React from 'react'
```

Comment:
Can you add a comment here, so we know it's an open issue, and could be rollback once storybook fix that?